File: 3.선그리기/solution(기섭).cpp
#include <iostream>
using namespace std;
 
int main(){
    int N, sum=0,start, end, arr[10001]={0};
    cin >> N;
    for(int i=0;i<N;i++){
        cin >> start >> end;
        for(int j=start; j<end; j++)
            arr[j]=1;
    }
    for(int i=1;i<10000;i++)
        if(arr[i])sum++;
    cout << sum;
 
}

File: 3.선그리기/solution(기섭).py
arr = [ 0 for i in range(10001) ]
N=input()
 
for i in range(1,int(N)+1):
    start,end = input().split()
    for j in range ( int(start), int(end) ):
        arr[j]=1
sum=0
for i in range(1, 10001):
    if arr[i]==1:
        sum+=1
print(sum)